Formula/go-jira.rb

```diff
@@ -0,0 +1,31 @@
+class GoJira < Formula
+  desc "Simple jira command-line client in Go"
+  homepage "https://github.com/Netflix-Skunkworks/go-jira"
+  url "https://github.com/Netflix-Skunkworks/go-jira/archive/v1.0.12.tar.gz"
+  sha256 "7ce5d6dcc5021f2019b0107c4ad342979d18bb110188b810017b2d94ff384a33"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/gopkg.in/Netflix-Skunkworks/go-jira.v1"
+    dir.install buildpath.children
+
+    ldflags = ["-w", "-s"]
```

Comment:
What exactly are we gaining here by passing `ldflags.join(" ")` instead of just `"-ldflags", "-w -s"`...?

Reply:
in a previous iteration I was building this list w/some conditionals. will update